packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts

```diff
@@ -616,4 +564,81 @@ function preparePayloadAttributes(
   return payloadAttributes;
 }
 
-/** process_sync_committee_contributions is implemented in syncCommitteeContribution.getSyncAggregate */
+export async function produceCommonBlockBody<T extends BlockType>(
+  this: BeaconChain,
+  blockType: T,
+  currentState: CachedBeaconStateAllForks,
+  {
+    randaoReveal,
+    graffiti,
+    slot,
+    parentSlot,
+    parentBlockRoot,
+  }: BlockAttributes & {
+    parentSlot: Slot;
+    parentBlockRoot: Root;
+  }
+): Promise<CommonBlockBody> {
+  const stepsMetrics =
+    blockType === BlockType.Full
+      ? this.metrics?.executionBlockProductionTimeSteps
+      : this.metrics?.builderBlockProductionTimeSteps;
```

Comment:
Is it still required to have 2 different metrics for this? maybe should be consolidated into a single metric

There is also the option now to return blinded local blocks (https://github.com/ChainSafe/lodestar/pull/6227), metric naming should reflect that.

Reply:
Mentioned it earlier in discussion that this metric need cleanup.